olm-catalog/serverless-operator/project.yaml

```diff
@@ -4,20 +4,20 @@ project:
     # all components in `dependencies.previous` to the same versions as `dependencies` in the same PR.
     # Otherwise, the upgrade tests will not pass, as we have a different SO version with the same bundle contents.
     # Also make sure to update values under `olm.previous` by copying from `olm.replaces` and `olm.skipRange`.
-    version: 1.32.0
```

Comment:
isnt the 1.32 branch for ... 1.32 version? :sweat_smile:

Reply:
sorry, I got confused by the prev. title of the PR :sweat_smile: 

All cool here!